public/datasets/erin_candy_preferences.json

```diff
@@ -0,0 +1,59 @@
+{
+  "fields": [
+  	{
+  		"id": "name",
+  		"type": "categorical",
+  		"name": "Name"
```

Comment:
I need to go through and set proper types for each of these.

I added a friendly name field, but haven't set it yet.  Do we think it will be useful?

Reply:
I think the friendly name field will be useful, especially for more complicated datasets or when the column name isn't very descriptive. However, I don't think we should worry about setting the friendly name for all the columns for each of these datasets since we likely won't end up using this data.

Reply:
Also, might be interesting to loop Brent in on the general idea of how this metadata will work (once you're feeling like it's stable-ish and ready to share) since I suspect he'll be working on writing up these metadata files for the final sets?